### ISSUE TITLE
When AutomaticKeepAliveClientMixin used SnackBar lagging.

### ISSUE THREAD

**Issue by vlytvyne** [flutter/flutter#59675]
When I use `AutomaticKeepAliveClientMixin` with `wantKeepAlive = true` and call `showSnackBar()` from the state object the snackbar is shown but in laggy way, it continues to pop up several times whether you try to dismiss it or just wait. SnackBar is displayed as intended if `AutomaticKeepAliveClientMixin` not used.

![bug](https://user-images.githubusercontent.com/44924680/84891400-7ad31380-b0a4-11ea-90c0-df8be92641cd.gif)

**Expected results:** 
Snackbar is shown only once and won't pop up be himself if user dismiss it.

**Actual results:** 
Snackbar is shown several times, and if user tries to dismiss it, it pops up again.

Flutter doctor:
```
[✓] Flutter (Channel stable, v1.17.1, on Mac OS X 10.15.4 19E287, locale uk-UA)
    • Flutter version 1.17.1 at /Users/vlytvyne/flutter
    • Framework revision f7a6a7906b (5 weeks ago), 2020-05-12 18:39:00 -0700
    • Engine revision 6bc433c6b6
    • Dart version 2.8.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/vlytvyne/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.2

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] IntelliJ IDEA Community Edition (version 2019.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 10 (API 29) (emulator)

! Doctor found issues in 1 category.
```

**Comment by VladyslavBondarenko** [comment]
Hi @vlytvyne 
could you provide a complete minimal reproducible code sample?

**Comment by iapicca** [comment]
Without additional information we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
